cms/envs/common.py

```diff
@@ -916,6 +916,9 @@
     # django-oauth-toolkit
     'oauth2_provider',
 
+    # django-oauth-plus
+    'oauth_provider',
```

Comment:
I'm not quite sure what this change does - it doesn't seem to be related to the warnings I see (when running studio via master), and the no-warnings behavior observed from the fix remains if I remove this INSTALLED_APP.